packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1233,10 +1237,13 @@ export class ContainerRuntime
 			loader,
 			pendingLocalState,
 			supportedFeatures,
+			snapshotWithContents,
 		} = context;
 
 		this.innerDeltaManager = deltaManager;
 		this.deltaManager = new DeltaManagerSummarizerProxy(this.innerDeltaManager);
+		const baseSnapshotTree =
```

Comment:
I'd suggest finding better names. baseSnapshot & baseSnapshotTree - not very clear from names how they are different.
Maybe baseSnapshotTree  -> snapshotWithContent?

Reply:
Removed as per other comments.